src/librustdoc/passes.rs

```diff
@@ -242,6 +239,21 @@ impl<'a> fold::DocFolder for ImplStripper<'a> {
     }
 }
 
+// This stripper discards all private import statements (`use`, `extern crate`)
+struct ImportStripper;
+impl fold::DocFolder for ImportStripper {
+    fn fold_item(&mut self, i: Item) -> Option<Item> {
+        match i.inner {
+            clean::ExternCrateItem(..) |
```

Comment:
I think the pattern guard on the next line should take care of that (?) :smiley: 

Or do you mean a rustdoc test? I've added one since there doesn't appear to be any rustdoc test for `pub extern crate`.